src/serialmonitor/serialMonitor.ts

```diff
@@ -38,256 +30,170 @@ export class SerialMonitor implements vscode.Disposable {
 
     private static _serialMonitor: SerialMonitor = null;
 
-    private _currentPort: string;
-
-    private _currentBaudRate: number;
+    private serialMonitorApi: SerialMonitorApi | undefined;
+    private extensionContext: vscode.ExtensionContext;
+    private currentPort: string;
+    private activePort: Port | undefined;
+    private lastSelectedBaudRate: number = 9600; // arbitrary default.

```

Comment:
Is this the default baud rate for Arduino's own first-party serial monitor? If so, let's document it that way instead of saying it's arbitrary, because we should probably follow Arduino's lead to reduce surprise.

Reply:
Changing.